maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -142,32 +142,32 @@ public void validateRawModel( Model m, ModelBuildingRequest request, ModelProble
                 validateStringNotEmpty( "version", problems, Severity.FATAL, Version.V20, m.getVersion(), m );
             }
 
-            validate20RawDependencies( problems, m.getDependencies(), "dependencies.dependency", request );
+            validate20RawDependencies( problems, m.getDependencies(), "dependencies.dependency", "", request );
```

Comment:
I think these cases should use `StringUtils.EMPTY` to denote that an empty string has been intentionally used.

Reply:
Sure, I'll push a commit to do that.